weave-js/src/components/Panel2/ChildPanelExportReport/ChildPanelExportReport.tsx

```diff
@@ -1,15 +1,38 @@
-import React from 'react';
+import React, {useState} from 'react';
 import _ from 'lodash';
 
 import {Alert} from '../../Alert.styles';
 import {Button} from '../../Button';
 import {Tailwind} from '../../Tailwind';
 import {useCloseDrawer, useSelectedPath} from '../PanelInteractContext';
+import {ReportSelection} from './ReportSelection';
+import {ChildPanelFullConfig} from '../ChildPanel';
+import {EntityOption, ReportOption} from './utils';
 
-export const ChildPanelExportReport = () => {
+type ChildPanelExportReportProps = {
+  config: ChildPanelFullConfig;
+};
+
+export const ChildPanelExportReport = ({
+  config,
+}: ChildPanelExportReportProps) => {
   const selectedPath = useSelectedPath();
   const closeDrawer = useCloseDrawer();
 
+  const [selectedEntity, setSelectedEntity] = useState<EntityOption | null>(
+    null
+  );
+  const [selectedReport, setSelectedReport] = useState<ReportOption | null>(
+    null
+  );
```

Comment:
Added these selected values in the parent component because we'll need them when user clicks on `Add panel`